src/twisted/test/test_rebuild.py

```diff
@@ -43,55 +45,49 @@ class RebuildTests(unittest.TestCase):
     def setUp(self):
         self.libPath = self.mktemp()
         os.mkdir(self.libPath)
-        self.fakelibPath = os.path.join(self.libPath, 'twisted_rebuild_fakelib')
+        self.fakelibPath = os.path.join(self.libPath,
+                                        'twisted_rebuild_fakelib')
         os.mkdir(self.fakelibPath)
         open(os.path.join(self.fakelibPath, '__init__.py'), 'w').close()
         sys.path.insert(0, self.libPath)
 
+
     def tearDown(self):
         sys.path.remove(self.libPath)
 
-    def testFileRebuild(self):
+
+    def test_FileRebuild(self):
         from twisted.python.util import sibpath
         import shutil, time
         shutil.copyfile(sibpath(__file__, "myrebuilder1.py"),
                         os.path.join(self.fakelibPath, "myrebuilder.py"))
         from twisted_rebuild_fakelib import myrebuilder
         a = myrebuilder.A()
-        try:
-            object
-        except NameError:
+        from twisted.test import test_rebuild
+        b = myrebuilder.B()
+        class C(myrebuilder.B):
             pass
-        else:
-            from twisted.test import test_rebuild
-            b = myrebuilder.B()
-            class C(myrebuilder.B):
-                pass
-            test_rebuild.C = C
-            C()
+        test_rebuild.C = C
+        C()
         i = myrebuilder.Inherit()
         self.assertEqual(a.a(), 'a')
-        # necessary because the file has not "changed" if a second has not gone
+        # Necessary because the file has not "changed" if a second has not gone
         # by in unix.  This sucks, but it's not often that you'll be doing more
         # than one reload per second.
         time.sleep(1.1)
         shutil.copyfile(sibpath(__file__, "myrebuilder2.py"),
                         os.path.join(self.fakelibPath, "myrebuilder.py"))
         rebuild.rebuild(myrebuilder)
-        try:
-            object
-        except NameError:
-            pass
-        else:
-            b2 = myrebuilder.B()
-            self.assertEqual(b2.b(), 'c')
-            self.assertEqual(b.b(), 'c')
+        b2 = myrebuilder.B()
+        self.assertEqual(b2.b(), 'c')
+        self.assertEqual(b.b(), 'c')
         self.assertEqual(i.a(), 'd')
         self.assertEqual(a.a(), 'b')
-        # more work to be done on new-style classes
+        # More work to be done on new-style classes
         # self.assertEqual(c.b(), 'c')
```

Comment:
I think the test is unfinished because I think the intention was that after doing  `rebuild.rebuild(myrebuilder)`, then `class C(myrebuilder.B)`, would inherit from the rebuilt `myrebuilder.B`, and then `C.b()` would return `'c'` instead of `'b'`.

However it looks like this feature was not finished.
Finishing this off is outside the scope of this pull request.